contracts/peripheral/grants/AuraArbBalGrant.sol

```diff
@@ -0,0 +1,263 @@
+        _;
+    }
+
+    /**
+     * @notice Modifier that only allows something to be called when the contract is inactive
+     */
+    modifier whileInactive() {
+        require(cooldownStart != 0 && block.timestamp > cooldownStart + cooldownPeriod, "active");
+        _;
+    }
+
+    /**
+     * @notice Modifier that only allows something to be called when the contract is active
+     */
+    modifier whileActive() {
+        require(cooldownStart == 0, "!active");
+        _;
+    }
+
+    /* ----------------------------------------------------------------
+       Init 
+    ---------------------------------------------------------------- */
+
+    /**
+     * @dev Initialise the contract values
+     * @param _aura        AURA token
+     * @param _bpt         BPT token
+     * @param _poolId      poolID of the 8020 pool
+     * @param _tokenOrder  Order of AURA, BAL, ARB in the pool
+     */
+    function init(
+        IERC20 _aura,
+        IERC20 _bpt,
+        bytes32 _poolId,
+        uint256[3] memory _tokenOrder
+    ) external onlyAuth {
+        require(address(AURA) == address(0), "already initialized");
+
+        AURA = _aura;
+        BPT = _bpt;
+        POOL_ID = _poolId;
+
+        for (uint256 i; i < 3; i++) {
+            tokenOrder[i] = _tokenOrder[i];
+        }
+
+        _aura.safeApprove(address(BALANCER_VAULT), type(uint256).max);
+        BAL.safeApprove(address(BALANCER_VAULT), type(uint256).max);
+        ARB.safeApprove(address(BALANCER_VAULT), type(uint256).max);
+    }
+
+    /* ----------------------------------------------------------------
+       Shared Functions
+    ---------------------------------------------------------------- */
+
+    /**
+     * @notice Sends BAL and ARB to balancer and AURA to project
+     * @dev grant must be inactive in order for this to be called
+     */
+    function withdrawBalances() external onlyAuth whileInactive {
+        // Send AURA to project msig
+        uint256 auraBalance = AURA.balanceOf(address(this));
+        AURA.safeTransfer(project, auraBalance);
+        // Send BAL and ARB to balancer msig
+        uint256 balBalance = BAL.balanceOf(address(this));
+        BAL.safeTransfer(balancer, balBalance);
+        uint256 arbBalance = ARB.balanceOf(address(this));
+        ARB.safeTransfer(balancer, arbBalance);
+
+        emit WithdrawBalances(auraBalance, balBalance, arbBalance);
+    }
+
+    /**
+     * @notice Join the pool
+     * @dev Only callable by an authenticated party
+     * @dev Only callable when active
+     * @param _minAmountOut Min amount of BPT to get out
+     */
+    function join(uint256 _minAmountOut) external onlyAuth whileActive {
+        _joinPool(_minAmountOut);
+    }
+
+    /**
+     * @notice Allows auth to start cooldown timer
+     */
+    function startCooldown() external onlyAuth {
+        cooldownStart = block.timestamp;
+        emit StartCooldown(block.timestamp, block.timestamp + cooldownPeriod);
+    }
```

Comment:
We should add a require here to make sure the ball can't just be continuously kicked down the road if it's already been started.
```suggestion
    function startCooldown() external onlyAuth {
        require(cooldownStart == 0, "Already Cooldown");
        cooldownStart = block.timestamp;
        emit StartCooldown(block.timestamp, block.timestamp + cooldownPeriod);
    }
```